webApp/src/main/java/com/infoshare/controller/VolunteerController.java

```diff
@@ -28,9 +32,23 @@ public String searchForVolunteer(Model model) {
         return getTestViewWithPageUnderConstructionMessage(model);
     }
 
+    @PostMapping("/submit-edited-form")
+    public String submitEditedVolunteerForm(@Valid @ModelAttribute("volunteerForm") VolunteerForm volunteerForm, BindingResult br, Model model) {
+        if (br.hasErrors()) {
+            model.addAttribute("types", volunteerService.getTypesOfHelp());
+            return "edited-volunteer-form";
+        } else {
+            volunteerService.editVolunteerData(volunteerForm.getName(), volunteerForm.getLocation(), volunteerForm.getEmail(), volunteerForm.getPhone(), volunteerForm.getTypeOfHelp(), volunteerForm.getIsAvailable(), volunteerForm.getUuid());
+            return "successfully-edited-form";
+        }
+    }
+
     @GetMapping("/edit")
```

Comment:
Przydałby się jakiś widok, gdzie wpiszemy adres email wolontariusza, którego chcemy edytować. Albo może jakiś widok listy wolontariuszy, a przy każdym przycisk do edycji 😜 
Ale nie wiem, może były jakieś ustalenia w zespole jak będzie wyglądało wybieranie wolontariusza do edycji.

Reply:
To też zapytam na projekcie :)

Reply:
dodałam oddzielny widok, gdzie sprawdza czy mail jest w bazie danych